src/modules/roc_audio/profiler.h

```diff
@@ -37,15 +37,14 @@ class Profiler : public core::NonCopyable<> {
     //! Initialization.
     Profiler(core::IAllocator& allocator,
              packet::channel_mask_t channels,
-             size_t sample_rate);
+             size_t sample_rate,
+             core::nanoseconds_t interval = LogInterval);
 
     //! Init audio frame.
     void begin_frame(size_t frame_size);
 
     //! Profile frame speed
-    void end_frame(size_t frame_size,
-                   core::nanoseconds_t elapsed,
-                   const core::nanoseconds_t interval = LogInterval);
+    void end_frame(size_t frame_size, core::nanoseconds_t elapsed);
 
     //! For Testing Only
     double get_moving_avg() {
```

Comment:
can this be implemented in header or should I move this implementation into the cpp

Reply:
I think both is OK.